website/build.gradle

```diff
@@ -230,11 +264,11 @@ publishWebsite.dependsOn commitWebsite
  *   ./gradlew :beam-website:stageWebsite -PgithubPullRequestId=${ghprbPullId} -PwebsiteBucket=foo
  */
 task stageWebsite << {
-  assert project.hasProperty('githubPullRequestId')
-  assert githubPullRequestId.isInteger()
+  // TODO: Refactor this duplication with build task
+  def baseUrl = project.findProperty('githubPullRequestId') ? "${project.findProperty('githubPullRequestId')}" : 'latest'
```

Comment:
Not sure this works ... I'm seeing the findProperty succeeding in non-PR jobs with the value remaining the uninterpolated value, --baseurl=/${ghprbPullId}

Reply:
This works in the latest version. I ran `./gradlew :beam-website:stageWebsite` locally: [Gradle Build Scan](https://gradle.com/s/rmd2mxqfbrofg)

```
Website published to http://apache-beam-website-pull-requests.storage.googleapis.com/latest/index.html
```